flags.go

```diff
@@ -47,3 +49,38 @@ func (s *noSummaryValue) String() string {
 	// flip all the bits, since the flag value is the negative of what is stored
 	return (testjson.SummarizeAll ^ s.value).String()
 }
+
+var junitFieldFormatValues = "full, relative, short"
+
+type junitFieldFormatValue struct {
+	value junitxml.FormatFunc
+}
+
+func (f *junitFieldFormatValue) Set(val string) error {
+	switch val {
+	case "full":
+		return nil
+	case "relative":
+		f.value = testjson.RelativePackagePath
+		return nil
+	case "short":
+		f.value = path.Base
+		return nil
+	}
+	return errors.Errorf("invalid value: %v, must be one of: "+junitFieldFormatValues, val)
+}
+
+func (f *junitFieldFormatValue) Type() string {
+	return "field-format"
+}
+
+func (f *junitFieldFormatValue) String() string {
+	return "full"
+}
```

Comment:
Is it ok, that in any case `full` is returned here?

Reply:
Yup, this is what gets printed as the default value. Since there is no way to change the default value, this should always be correct.